pkg/scheduler/api/types.go

```diff
@@ -171,3 +171,6 @@ type VictimTasksFn func() []*TaskInfo
 
 // AllocatableFn is the func declaration used to check whether the task can be allocated
 type AllocatableFn func(*QueueInfo, *TaskInfo) bool
+
+// VictimsFromCandidatesFn is the func declaration used to select victim tasks from candidates and evict them
+type VictimsFromCandidatesFn func([]*TaskInfo) []*TaskInfo
```

Comment:
Why not reuse `VictimTasksFn`?

Reply:
`VictimTaskFn` is a function without input parameters, which is not so suitable for shuffle action. In shuffle action, the input parameter should be a collection of running tasks.

Reply:
That should be ok to update `victimTasksFn` for that.

Reply:
+1 for resuing VictimTasksFn

Reply:
OK. Let me give a upate.